src/components/StreakImage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import {
+    StyleSheet,
+    Text,
+    ImageBackground
+} from 'react-native';
+
+import {
+    View
+} from 'native-base';
+
+import {
+    Streak
+} from '../components'
+
+export class StreakImage extends React.Component {
```

Comment:
When we aren't managing state usually we want a pure component for optimization. So in this case I would make this class into a function component. And even if we need to manage state, we can use react hooks.